index.bs

```diff
@@ -2547,17 +2547,17 @@ Proposed Changes</h4>
 	<dfn lt="proposed change">proposed changes</dfn>.
 
 <h4 id="contributor-license">
-Contributor License Grants</h4>
+License Grants from Non-Participants</h4>
 
-	When a contributor who is not already obligated under the Patent Policy
+	When a party who is not already obligated under the Patent Policy
 	offers a change in class 3 or 4
 	(as described in [[#correction-classes]]) to a technical report under this process
-	the [=Team=] <em class=rfc2119>must</em> request from the contributor
+	the [=Team=] <em class=rfc2119>must</em> request from them
 	a recorded royalty-free patent commitment;
 	for a change in class 4, the Team <em class=rfc2119>must</em> secure such commitment.
 	Such commitment <em class=rfc2119>should</em> cover,
 	at a minimum,
-	all the contributor's Essential Claims both in the contribution,
+	all the party's Essential Claims both in the contribution,
```

Comment:
s/the party's// ?

Reply:
No, because we can't obligate them for essential claims they don't have. :)